Web/Edubase.Web.UI/Web.config

```diff
@@ -245,6 +245,14 @@
         <assemblyIdentity name="Microsoft.Bcl.AsyncInterfaces" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Glimpse.AspNet" publicKeyToken="null" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.9.2.0" newVersion="1.9.2.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Glimpse.Core" publicKeyToken="null" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.8.6.0" newVersion="1.8.6.0" />
+      </dependentAssembly>
```

Comment:
Where have the changes to the web.config come from?

Reply:
I'm not exactly sure. My guess is 'Slug.Hasher' which is a nugget package I was going to use. I would look into it

Reply:
I have looked into this and it appears the project once used Glimpse for measuring performance.
Here is a bit about Glimpse: https://learn.microsoft.com/en-us/aspnet/mvc/overview/performance/profile-and-debug-your-aspnet-mvc-app-with-glimpse